mu4/notation/qml/MuseScore/NotationScene/ViewModeControl.qml

```diff
@@ -0,0 +1,31 @@
+import QtQuick 2.7
+import MuseScore.NotationScene 1.0
+import MuseScore.UiComponents 1.0
+import MuseScore.Ui 1.0
+
+Row {
```

Comment:
I bet you don't need row here, StyledComboBox is a good candidate for the "root" item